README.rst

```diff
@@ -7,8 +7,9 @@
 • `Chat (Community & Support) <https://dvc.org/chat>`_
 • `Tutorial <https://dvc.org/doc/get-started>`_
 • `Mailing List <https://sweedom.us10.list-manage.com/subscribe/post?u=a08bf93caae4063c4e6a351f6&id=24c0ecc49a>`_
+• `VS Code Extension <https://marketplace.visualstudio.com/items?itemName=Iterative.dvc>`_
 
-|CI| |Maintainability| |Coverage| |Donate| |DOI|
+|CI| |Maintainability| |Coverage| |Donate| |VSCode| |DOI|
```

Comment:
VS Code

Reply:
This doesn't accept whitespace. It's not visible anyway on the page, so I don't think it's an issue. Let me know if you think it's a problem.